src/recipes/Healthy-salad-with-baked-goat-cheese.md

```diff
@@ -0,0 +1,55 @@
+---
+tags: ["Salad", "Cheese", "Peach"]
```

Comment:
hi, can you just have this as one of the categories - Salad is perfect, so remove 'cheese' and 'peach' please.